components/popconfirm/index.en-US.md

```diff
@@ -16,13 +16,13 @@ The difference with the `confirm` modal dialog is that it's more lightweight tha
 
 | Param | Description | Type | Default value |
 | --- | --- | --- | --- |
-| cancelText | text of the Cancel button | string | `Cancel` |
-| okText | text of the Confirm button | string | `OK` |
+| cancelText | text of the Cancel button | string\|ReactNode\|() => ReactNode | `Cancel` |
+| okText | text of the Confirm button | string\|ReactNode\|() => ReactNode | `OK` |
 | okType | Button `type` of the Confirm button | string | `primary` |
-| title | title of the confirmation box | string\|ReactNode | - |
+| title | title of the confirmation box | string\|ReactNode\|() => ReactNode | - |
 | onCancel | callback of cancel | function(e) | - |
 | onConfirm | callback of confirmation | function(e) | - |
-| icon | customize icon of confirmation | ReactNode | `<ExclamationCircle />` |
+| icon | customize icon of confirmation | ReactNode\|() => ReactNode | `<ExclamationCircle />` |
```

Comment:
`title` is enough. I perfer to keep `okText`, `cancelText` and `icon` as ReactNode unless more requirements coming.

Reply:
Sure, will make the amendments 👍

Reply:
Done